guides/doc-Content_Management_Guide/topics/Using_ISS.adoc

```diff
@@ -100,7 +100,9 @@ This option is available for all content-export operations. You can use the `des
 === Exporting a Content View Version
 
 You can export a version of a Content View to an archive file from {ProjectServer} and use this archive file to create the same Content View version on another {ProjectServer} or on another {ProjectServer} organization.
-{Project} does not export composite Content Views.
+{Project} exports composite Content Views as normal content views.
+The composite nature is not retained.
+On importing the exported archive, a normal content-view will get created/updated in the downstream {ProjectServer}.
```

Comment:
@melcorr you ok with this ^ ?

Reply:
The word "normal" seems strange.

Reply:
Oh damn I edited your comment rather than quote @parthaa - I'm an idiot.